spec/index.html

```diff
@@ -786,18 +786,8 @@ <h3>Initial Text Direction</h3>
         so that they are visually ordered as expected.
         However, this is not sufficient to correctly render bidirectional text.</p>
 
-      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" (for Hebrew) may be displayed
-        incorrectly unless the <a>base direction</a> of `rtl` (for right-to-left)
-        is also provided.
-        When this is provided to a user agent,
-        for example using the HTML `dir` attribute,
-        it would be correctly presented as follows: <div lang="he" dir="rtl">פעילות הבינאום, W3C</div></p>
-
-      <p class="note">The absence of a <a>base direction</a> does not necessarily imply that
-        the text has no initial text direction;
-        as described in [[[?UAX9]]],
-        strings may be embedded within structures which establish an <em>embedding direction</em>,
-        which determines the default bidirectional orientation of text.</p>
+<p>For example, some text with a language tag "he" might be displayed in a left-to-right context (such as an English web page) as <bdi dir=ltr>פעילות הבינאום, W3C</bdi>, which is incorrect. When provided to a user agent including base direction information (such as using HTML's `dir` attribute) it can then be correctly presented as:</p>
+<div lang="he" dir="rtl">פעילות הבינאום, W3C</div>
```

Comment:
```suggestion
       <p>For example, some text with a language tag "`he`" might be displayed
         in a left-to-right context (such as an English web page) as
         <bdi dir=ltr>פעילות הבינאום, W3C</bdi>, which is incorrect. When provided
         to a user agent including base direction information (such as using
         HTML's `dir` attribute) it can then be correctly presented as:
       </p>
       <div lang="he" dir="rtl">פעילות הבינאום, W3C</div>
```